src/renderer/components/password-settings/password-settings.js

```diff
@@ -0,0 +1,64 @@
+import Vue from 'vue'
+import { mapActions } from 'vuex'
+import FtSettingsSection from '../ft-settings-section/ft-settings-section.vue'
+import FtSelect from '../ft-select/ft-select.vue'
+import FtInput from '../ft-input/ft-input.vue'
+import FtToggleSwitch from '../ft-toggle-switch/ft-toggle-switch.vue'
+import FtFlexBox from '../ft-flex-box/ft-flex-box.vue'
+import FtButton from '../ft-button/ft-button.vue'
+import FtPrompt from '../ft-prompt/ft-prompt.vue'
+
+export default Vue.extend({
+  name: 'PasswordSettings',
+  components: {
+    'ft-settings-section': FtSettingsSection,
+    'ft-select': FtSelect,
+    'ft-input': FtInput,
+    'ft-toggle-switch': FtToggleSwitch,
+    'ft-flex-box': FtFlexBox,
+    'ft-button': FtButton,
+    'ft-prompt': FtPrompt,
```

Comment:
Same for this file.

Reply:
Sorry, that was a copy and paste error on my part.